seonshine_frontend/src/pages/dashboard/index.tsx

```diff
@@ -47,66 +52,129 @@ const Dashboard = () => {
+      </Box>
+    </>
+  );
+
+  const renderUserBoxes = () => (
+    <>
+      <Box className="w-full md:w-1/4 flex flex-col bg-white rounded-md p-4">
+        <Stack className="flex-grow">Today's Restaurant</Stack>
+        <Stack className="font-bold text-2xl">{dashboardSummary?.today_restaurant_name}</Stack>
+      </Box>
+      <Box className="w-full md:w-1/4 flex flex-col bg-white rounded-md p-4">
+        <Stack className="flex-grow">Order Status</Stack>
+        <Stack className="self-end font-bold text-2xl">
+          {dashboardSummary?.current_order_status == 1 ? 'Ordered' : 'Not Ordered'}
+        </Stack>
+      </Box>
+      <Box className="w-full md:w-1/4 flex flex-col bg-white rounded-md p-4">
+        <Stack className="flex-grow">Ordered Item</Stack>
+        <Stack className="self-end font-bold text-2xl">{dashboardSummary?.current_order_item_name}</Stack>
+      </Box>
+      <Box className="w-full md:w-1/4 flex flex-col bg-white rounded-md p-4">
+        <Stack className="flex-grow">Ordered Users</Stack>
+        <Stack className="self-end font-bold text-2xl">{dashboardSummary?.today_order_users_count}</Stack>
+      </Box>
+    </>
+  );
+
+  const renderRestaurantBoxes = () => (
+    <>
+      <Box className="w-full md:w-1/4 flex flex-col bg-white rounded-md p-4">
+        <Stack className="flex-grow">Assigned Date</Stack>
+        <Stack className="font-bold text-2xl">
+          {dashboardSummary?.assigned_weekdays?.map((day) => dayMapper(day)).join(', ')}
+        </Stack>
+      </Box>
+      <Box className="w-full md:w-1/4 flex flex-col bg-white rounded-md p-4">
+        <Stack className="flex-grow">Order Status</Stack>
+        <Stack className="self-end font-bold text-2xl">{isOrderEnabled() ? 'Waiting' : 'Ordered'}</Stack>
+      </Box>
+    </>
+  );
 
   return (
     <Box className="px-4 py-2 md:px-8 md:py-4">
       <Stack className="flex flex-wrap md:flex-nowrap gap-4 md:gap-6">
-        <Box className="w-full md:w-1/4 flex flex-col bg-white rounded-md p-4">
-          <Stack className="flex-grow">Today's Restaurant</Stack>
-          <Stack className="font-bold text-2xl">PAPA'S CHICKEN</Stack>
-        </Box>
-        <Box className="w-full md:w-1/4 flex flex-col bg-white rounded-md p-4">
-          <Stack className="flex-grow">Ordered Users</Stack>
-          <Stack className="self-end font-bold text-2xl">12</Stack>
-        </Box>
-        <Box className="w-full md:w-1/4 flex flex-col bg-white rounded-md p-4">
-          <Stack className="flex-grow">Active Users</Stack>
-          <Stack className="self-end font-bold text-2xl">32</Stack>
-        </Box>
-        <Box className="w-full md:w-1/4 flex flex-col bg-white rounded-md p-4">
-          <Stack className="flex-grow">Waiting for Approval</Stack>
-          <Stack className="self-end font-bold text-2xl">8</Stack>
-        </Box>
+        {currentUser?.role_id === RoleEnum.ADMIN && renderAdminBoxes()}
```

Comment:
please define new function for this check. Ex: `renderHeaderSummaryBoxesBasedRole`